salt/example.top

```diff
@@ -4,6 +4,7 @@ base:
         - elife.redis-server
         - elife.docker
         - elife-bot.strip-coverletter
+        - elife-bot.email-templates
```

Comment:
it doesn't have to live in a separate state file. If elife-bot needs it then it can live just as happily in `init.sls`

Reply:
you'll need to update the top file in builder-configuration as well.

Reply:
https://github.com/elifesciences/builder-configuration/blob/master/salt/top.sls#L63-L92

Reply:
I could avoid changing that if I just add this to the `init.sls`?

Reply:
yes

Reply:
we typically split them into logical units when different environments need different things, but if all environments need it then just add it to init.sls